cypress/support/commands.ts

```diff
@@ -2,21 +2,42 @@ import '@testing-library/cypress/add-commands';
 
 // Add Custom commands here and their types in `./index.d.ts`
 
-Cypress.Commands.add('start', (path: string) => {
-  cy.viewport('macbook-13');
+Cypress.Commands.add('start', (path: string, withLogin: boolean = false) => {
   cy.visit(path);
 
-  cy.findByLabelText(/email address/i).type(Cypress.env('username'));
-  cy.findByLabelText(/password/i).type(Cypress.env('password'));
-  cy.get('button[type="submit"]').click();
+  if (withLogin) {
+    cy.get('#username').type(Cypress.env('username'));
+    cy.get('#password').type(Cypress.env('password'));
+    cy.get('button[type="submit"]').click();
+  }
 
-  cy.intercept('https://patient.qa.clin.ferlab.bio/patient/search').as('search-request');
-  cy.wait('@search-request');
+  cy.intercept('GET', 'https://patient.qa.clin.ferlab.bio/patient/search').as('search-request');
+  cy.wait('@search-request').its('request.headers.authorization').as('auth-token');
   cy.findByText('Patients').should('be.visible');
   return cy;
 });
 
 Cypress.Commands.add('logout', () => {
   cy.get('.userName > .ant-dropdown-trigger', { timeout: 10000 }).click()
     .get('.ant-dropdown-menu-item > :nth-child(1)', { timeout: 10000 }).click();
+
+  cy.findByText('Enter your credentials to log in').should('be.visible');
+  return cy;
+});
+
+Cypress.Commands.add('selectItem', (name: string, index: number) => {
+  cy.get(`#${name}_list + .rc-virtual-list .ant-select-item-option-content`)
+    .eq(index).click();
+  return cy;
 });
+
+Cypress.Commands.add('deletePatient', (patientId: string) => cy.get('@auth-token').then((token) => {
```

Comment:
This command doesn't work cause we need to delete all children first and it's really tough to have all the ids.

@YanLabCHU FHIR seems to have a "cascade" config we can turn on with a `cascade` param it might be dangerous tough. Maybe we could have a "clean" endpoint to use instead?

Reply:
Yes, FHIR offers the `cascade` option, but we won't use it (at least for now).
Cypress is running in the QA environment right now (and should preferably run in dev later). So, for now, just delete the patient resource (which you already do) to stop seeing it in the patient list. We don't have to worry about the other resources.

Reply:
Yes, FHIR offers the cascade option, but we won't use it (at least for now).
Cypress is running in the QA environment right now (and should preferably run in dev later). So, for now, just delete the patient resource (what you did here) to stop seeing it in the patient list. We don't have to worry about the other resources.

Reply:
You can ignore this step for now, I'll see what I can do later